docs/CHANGELOG.asciidoc

```diff
@@ -150,6 +150,9 @@ option. ({pull}30140[#29658])
 A new analysis plugin called `analysis_nori` that exposes the Lucene Korean
 analysis module.  ({pull}30397[#30397])
 
+Text fields now have an `index_phrases` parameter that allows faster phrase queries
```

Comment:
as a non native speaker it feels weird that `index_prefix` is singular while `index_phrases` is plural?